src/NHibernate/Type/GenericOrderedSetType.cs

```diff
@@ -11,6 +11,9 @@ namespace NHibernate.Type
 	[Serializable]
 	public class GenericOrderedSetType<T> : GenericSetType<T>
 	{
+		// ReSharper disable once StaticMemberInGenericType
```

Comment:
It tend to favor disabling globally such R# rules rather than putting R# specific comment in code. (Indeed I have ended up disabling so many rules that were too noisy at my previous work, that I have ended disabling wholly R#, and not missing it much.)